nixos/modules/services/continuous-integration/github-runner.nix

```diff
@@ -291,6 +291,17 @@ in
         RestrictRealtime = true;
         RestrictSUIDSGID = true;
         UMask = "0066";
+        ProtectProc = "invisible";
+        ProcSubset = "pid";
+        SystemCallArchitectures = "native";
```

Comment:
Actually running i686 binaries in a CI does seem common enough to allow it:

```suggestion
```

Reply:
Good point, done!